lib/nexpose/site.rb

```diff
@@ -232,6 +204,9 @@ def add_asset(asset)
       @assets << obj
     end
 
+    alias_method :add_host, :add_asset
```

Comment:
@gschneider-r7 @mdaines-r7 Should we deprecate the old forms (log a deprecation message and delegate to `Site#add_asset`) instead of aliasing them?